mage_ai/server/api/environments.py

```diff
@@ -0,0 +1,27 @@
+from .base import BaseHandler
+
+
+class ApiEnvironmentsHandler(BaseHandler):
```

Comment:
We should give the endpoint a better name. This endpoint is to manage the instances in a cluster instead of managing envrionments.